src/avram/migrator/migrator.cr

```diff
@@ -1,9 +1,8 @@
 module Avram::Migrator
   def self.run
     yield
-  rescue e : PQ::PQError
-    raise e.message.colorize(:red).to_s
-  rescue e : Exception
-    raise e.message.to_s
+  rescue e
+    puts e.inspect_with_backtrace
+    exit 1
```

Comment:
This is to be able to print the error with the cause but it means we are explicitly `exit`-ing in code